tests/ctst/features/cloudserverAuth.feature

```diff
@@ -5,9 +5,9 @@ Feature: AWS S3 Bucket operations
     @Cloudserver-Auth
     Scenario: Check Authentication on bucket object lock actions with Vault
         Given a IAM_USER type
-        And an IAM policy attached to the entity "user" with "Allow" effect to perform "CreateBucket" on "*"
-        And an IAM policy attached to the entity "user" with "<allow>" effect to perform "PutBucketObjectLockConfiguration" on "*"
-        And an IAM policy attached to the entity "user" with "<allow>" effect to perform "PutBucketVersioning" on "*"
+        And an IAM policy attached to the entity "user" with "Allow" effect to perform "s3" "CreateBucket" on "arn:aws:s3:::*"
+        And an IAM policy attached to the entity "user" with "<allow>" effect to perform "s3" "PutBucketObjectLockConfiguration" on "arn:aws:s3:::*"
+        And an IAM policy attached to the entity "user" with "<allow>" effect to perform "s3" "PutBucketVersioning" on "arn:aws:s3:::*"
         When the user tries to perform CreateBucket
```

Comment:
these tests updates go along with the Vault bump, which needs to land in 2.11 (patch version bump)
(i.e. depending on how we manage the scuba bump -cf earlier comment-, may need to split the PR)

Reply:
No these change are unrelated to Vault. The only part that is affected is https://github.com/scality/Zenko/pull/2214/commits/eb1e373b403206ec3477aaa2deaa0630e9bcaad3.

The other parts are only so that we can reuse the step with a different service name, so not s3, but sur.

Reply:
We can cherry pick the above commit if we want to bump Vault in 2.11 meanwhile.

Reply:
based on https://github.com/scality/Zenko/pull/2214#discussion_r2066219998, indeed may be better to split the vault-only part (and possibly refactoring to support SUR, so we potentially avoid conflicts down the line), which can be merged "immediately" on 2.11 ; and deal with the rest of this PR confidently, when SUR is available or we have created 2.12

Reply:
I created ZENKO-5000